pipeline/leech/main.go

```diff
@@ -139,6 +149,38 @@ func realMain(ctx context.Context) error {
 	return nil
 }
 
+func splitAndValidate(dottedTableName string) error {
```

Comment:
This is a fairly generic name for a function - split and validate what? At minimum, add godoc.

Reply:
Changed to `splitDatasetIDTableName`